extension/src/popup/components/signTransaction/Transaction/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { FlaggedKeys } from "types/transactions";
+
+import { Operations } from "popup/components/signTransaction/Operations";
+
+import "./styles.scss";
+
+interface TransactionProps {
+  flaggedKeys: FlaggedKeys;
+  isMemoRequired: boolean;
+  transaction: { [key: string]: any };
+}
+
+export const Transaction = ({
+  flaggedKeys,
+  isMemoRequired,
+  transaction,
+}: TransactionProps) => {
+  const { _operations } = transaction;
+
+  const operationText =
+    _operations && _operations.length > 1 ? "Operations:" : "Operation:";
+
+  return (
+    <div className="Transaction">
+      {_operations ? (
```

Comment:
If there are no `_operations` will the key not exist or be `null/undefined`?

Reply:
Hmmm good question - the original code was written for this a long time ago, I'll need to look into a use case where a properly formatted transaction has no operations 🤔